commander3/src/comm_nonlin_mod.f90

```diff
@@ -387,7 +389,7 @@ subroutine sample_specind_alm(cpar, iter, handle, comp_id, par_id)
           if (c%poltype(j) > 1 .and. cpar%only_pol .and. pl == 1) cycle 
 
           ! HKE -- disabling T for now
-          if (pl==1) cycle 
+          !if (pl==1) cycle 
```

Comment:
Why has this been reimplemented? This blocks alm spectral index sampling in total intensity

Reply:
This is a question for @hke